### PR TITLE
UTY-622 Enable alternative with no flag

### PR DESCRIPTION
**Pull request by firtina-improbable**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
Attempt to resolve issues raised by @zeroZshadow within #123 by removing the flag passed into the launch parameters, and force unique ids if no worker id is specified

#### Tests
Adjusted WorkerBaseTests to handle the parameter changes
- [x] Local Test runs
- [x] CI
- [x] Local Deployment
- [ ] Cloud Deployment

#### Documentation
Internal changes

#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.
@zeroZshadow @jessicafalk

### CODE REVIEW

File: workers/unity/Packages/com.improbable.gdk.core/Worker/WorkerBase.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ public abstract class WorkerBase : IDisposable
 
         public EntityGameObjectLinker EntityGameObjectLinker { get; private set; }
 
-        public bool UseDynamicId = false;
+        public bool UseDynamicId { get; protected set; }
```
Comment (zeroZshadow) [inline workers/unity/Packages/com.improbable.gdk.core/Worker/WorkerBase.cs]: What is this used for now?

Comment (jessicafalk) [reply workers/unity/Packages/com.improbable.gdk.core/Worker/WorkerBase.cs]: to check whether we need to generate a worker id or not. Look at line 34

Comment (firtina-improbable) [reply workers/unity/Packages/com.improbable.gdk.core/Worker/WorkerBase.cs]: 👍 to ☝️ , also to have the `UnityGameLogic` decide whether it wants to be unique per connection or not (as this is expected from the rest of SpatialOS)

**Comment by firtina-improbable** [comment]
✅ cloud with multiple clients

**Pull request merged**